core/sql/sqlcomp/CmpSeabaseDDLtable.cpp

```diff
@@ -12940,6 +12940,142 @@ TrafDesc *CmpSeabaseDDL::getSeabaseRoutineDescInternal(const NAString &catName,
 }
 
 
+short CmpSeabaseDDL::getSeabaseObjectComment(Int64 object_uid, 
```

Comment:
Should this be part of CmpSeabaseDDLcommentOn.cpp?  This keeps all comment related code in the same file.